codegen/src/main/scala/com/soundcloud/twinagle/codegen/ServerClientCodeGenerator.scala

```diff
@@ -11,8 +11,6 @@ import scalapb.options.compiler.Scalapb
 import scala.collection.JavaConverters._
 
 object ServerClientCodeGenerator extends protocbridge.ProtocCodeGenerator {
-  val params = scalapb.compiler.GeneratorParams()
```

Comment:
this was simply unused, right? not relevant to the changes?

Reply:
yes, it was unused. It confused me for a while because I thought I'd need to tweak it.

Reply:
yes, it was unused. It confused me for a while because I thought I'd need to tweak it.